cincinnati/src/plugins/mod.rs

```diff
@@ -398,8 +405,12 @@ where
         let tx = tx.clone();
 
         std::thread::spawn(move || {
-            let io_future =
-                async { tokio::time::timeout(timeout, process(plugins, initial_io)).await };
+            let span = get_tracer().start("plugins", None);
+            let io_future = async {
+                tokio::time::timeout(timeout, process(plugins, initial_io))
+                    .instrument(_span)
```

Comment:
Sorry, I had assumed you'd replace the other occurrences :smile: 

```suggestion
                    .instrument(span)
```